### ISSUE TITLE
iTunes 12.6.5 + Catalina 10.15.4 : no AppStore

### ISSUE THREAD

**Issue by MiniHarlock** [cormiertyshawn895/Retroactive#68]
Hi,
First, I installed iTunes 12.6.5 on Catalina a few weeks ago, thanks to Retroactive 1.3.
Really happy with the result !
Then I reinstalled it with Retroactive 1.5, even happier to see the Apps icons back.
But since Catalina 10.15.4 I can't update or download any app :-/
Following your instructions, I reinstalled iTunes with Retroactive 1.6 (and even 1.5), but I just get the same result…
I can see the apps which need to be updated, but when I launch any update process, even for one single app, the download is like cancelled right before starting. I don't get any error code.
I already spent so much time on this issue… I changed my Apple ID password, just in case, or payment infos. So I guess it's an issue with Retroactive + Catalina.
Are you aware of this issue or working on it ?
Thank you for your answer.

**Comment by cormiertyshawn895** [comment]
In April 2020, Apple has rolled out a server-side change that prevents all Mac users from purchasing and downloading iOS apps with iTunes. 

- You can use [Apple Configurator 2](https://apps.apple.com/app/apple-configurator-2/id1037126344) to [download iOS apps on your Mac](https://www.youtube.com/watch?v=M_3t06FEhR0).
- You can continue to use [iTunes 12.6.5](https://support.apple.com/HT208079) to purchase and download iOS apps on a Windows PC.

The Retroactive app and its README have been updated to recommend using [Apple Configurator 2](https://apps.apple.com/app/apple-configurator-2/id1037126344) instead.

![configurator-hint](https://user-images.githubusercontent.com/57056759/78525190-492bfb00-778b-11ea-8301-ebac281a3d64.jpg)

**Comment by MiniHarlock** [comment]
Thank you for you feedback.
Well, at this stage I really don't understand how this Apple Configurator 2 app works. Are we supposed to be able to manage our local apps library like we used to in iTunes ? Update apps and then sync with iPhone ?
I tried to modify the order of my apps, just for a test, clicked apply, AC2 told me it was done but no change on my iPhone. After that, I clicked on the Sign in enrollment Program (do I need to ?) but AC2 told me my Apple ID could not be enrolled ! I'm just so lost :-/
